### DISCUSSION TITLE
Dynamic VTX power due VRx RSSI ELRS backpack

### DISCUSSION THREAD

**Discussion by psicho100pro** [ExpressLRS/ExpressLRS#1480] [General]
Greetings to all lovers of flying.
In one of the many FPV forums, the user was asked about "dynamic VTX power" depending on the VRx RSSI.
This idea was immediately almost nailed to the cross by arguments that it is not possible, that it is not effective and that it is a waste of time to apply to a dying analog system.
I decided to interpret a similar question here.
In my opinion, backpacks modules could provide the mentioned function.
For example:
VRx_modif_firm_RSSI_output_stream - > EP82_Rxx => Espressif => ESP-xxx_Tx -> ELRS VTx_admin -> and so on 
or 
VRx_RSSI_pin -> ADS1015_I2C -> ESP-xxx => Espressif => ESP-xxx_Tx -> ELRS VTx_admin -> and so on
So what is your opinion? Is this idea just a trash?
Peace to all !!!

**Comment by commanderguy3001** [comment]
I like the idea, and it should be possible and probably not that hard to implement, the biggest problem would be to get the rssidbm out of the VRX I reckon.
One big problem I do see is that the switching up would have to be pretty fast to be effective, and it might take quite a long time for the VTX to be switched from like 25mw to 800mw, depending on how many power levels there are, as you probably don't want to switch from 25 to 800 right away.
This could indeed make the whole system at least kindof ineffective, as you might already be out of a noisy/RF blocking/interfering zone as soon as the power starts switching up.

**Comment by psicho100pro** [comment]
It is a pity that there are people who think they know everything.  
The question has already been answered. So we can look forward to. :)

![wez](https://user-images.githubusercontent.com/97060304/158010360-04768b9a-a1d4-4f53-b5e7-d555e992df8a.png)

**Reply by commanderguy3001** [reply]
> It is a pity that there are people who think they know everything. The question has already been answered. So we can look forward to. :)
> 
> ![wez](https://user-images.githubusercontent.com/97060304/158010360-04768b9a-a1d4-4f53-b5e7-d555e992df8a.png)

oh well that's interesting, I did not know that but it for sure answers the original question

**Reply by psicho100pro** [reply]
That sentence about people who know everything was not directed against you, if it sounded like that. I just for sure ;)
